packages/shared/components/ToolTip/ToolTip.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { Text, Popover } from 'design';
+import * as Icons from 'design/Icon';
+
+export function ToolTipInfo({ content }: { content: React.ReactNode }) {
```

Comment:
What's your thoughts on using `children` instead so we can have

```typescript jsx
<ToolTipInfo>
content here
</ToolTipInfo>
```

Reply:
![image](https://user-images.githubusercontent.com/43280172/210715006-ce7a4e8a-d153-4be6-8fc1-45847b29dcc9.png)